src/components/EventTag.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import PropTypes from 'prop-types'
+import hash from 'string-hash'
+
+const hotPink = css`
+  background-color: hsl(339, 87%, 55%);
+  color: white;
+`
+
+const blueish = css`
+  background-color: hsl(228, 57%, 58%);
+  color: white;
+`
+
+const yellow = css`
+  background-color: hsl(46, 100%, 68%);
+  color: black;
+`
+
+// we want consistent colors for tags we might not have seen
+const tagStyles = [hotPink, blueish, yellow]
+
+const knownValueStyles = {
+  Nightlife: yellow,
+  Music: blueish,
+  'Plays & Theatre': hotPink,
+}
+
+const selectTagStyle = value =>
+  knownValueStyles[value] || tagStyles[hash(value) % tagStyles.length]
+
+const EventTagSpan = styled.span`
+  display: inline-block;
+  border-radius: 4px;
+  padding: 0px 6px;
+  font-size: 14px;
+  line-height: 22px;
+  font-weight: 600;
+  font-family: Poppins, sans-serif;
+  ${({ style }) => style};
```

Comment:
is there a style component style guide? if there is does it say whether these style methods get pulled to the top of the css rule? it would make things easier to read if that was the case.

Reply:
I can do that. Personally I don't find it more or less difficult to read either way :)